contrib/build-push-ceph-container-imgs.sh

```diff
@@ -306,7 +309,11 @@ function create_registry_manifest {
 # MAIN #
 ########
 
-install_docker
+if [[ -x /usr/bin/dnf ]] ; then
+  install_podman
```

Comment:
we need to also s/docker/$DOCKER/ throughout and set DOCKER=podman here, right?

Reply:
See my comment below but I think installation podman-docker is enough for now because this package provide the docker command as a wrapper to podman

```console
# docker version
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
Version:       1.0.5
Go Version:    go1.11.6
OS/Arch:       linux/amd64
```

Reply:
much easier :)